client/stylesheets/_toCat.scss

```diff
@@ -4750,7 +4750,7 @@ On click the images will be displayed at normal size to complete the effect
 
 .notifMainWrap {
     width: 70%;
-    text-align: left;
+    margin:auto;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space